main.py

```diff
@@ -1,11 +1,36 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players_info = json.load(file)
+
+    for nick, info in players_info.items():
+        race_info = info["race"]
+        guild_info = info["guild"]
+
+        get_rase, _ = Race.objects.get_or_create(
+            name=race_info["name"],
+            description=race_info["description"])
+
+        if guild_info:
+            get_guild, _ = Guild.objects.get_or_create(
+                name=guild_info["name"],
+                description=guild_info["description"])
+        else:
+            get_guild = None
```

Comment:
You can create this viriable at the start and do not use else block .

If your condition is triggered, the contents of the variable will simply be overwritten